README.rst

```diff
@@ -546,6 +547,11 @@ It accepts an optional ``--migration-module`` flag indicating the django app
 name that should be passed to ``./manage.py makemigrations``, eg:
 ``./manage.py makemigrations $MIGRATION_MODULE``.
 
+``checkReST``
```

Comment:
let's call it `checkrst` to avoid people confusing it with the concept of REST API